components/entity-descriptions-display.vue

```diff
@@ -0,0 +1,34 @@
+<script lang="ts" setup>
+const t = useTranslations();
+
+const props = defineProps<{
+	descriptions: Array<{ value?: string }>;
+}>();
+
+const filteredDescriptions = computed(() => {
+	return props.descriptions.filter((description: { value?: string }) => {
+		return description.value;
+	});
+});
+</script>
+
+<template>
+	<template v-if="filteredDescriptions.length > 0">
+		<div
+			v-for="(description, index) in filteredDescriptions"
+			:key="description.value ?? 'descr' + index"
+		>
+			<template v-if="description.value">
+				<VDivider v-if="index > 0" />
```

Comment:
@Mocca101 can we replace this vuetify component with a plain `<div role="separator">`?

Reply:
Oh yes, sure!